### ISSUE TITLE
[BUG] - Carousel does not work with single pointer event on smaller touch devices

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2931]
### Description

If the carousel is used on a smaller touch device where the arrow buttons for navigation has been disappeared, it is only possible to navigate to next/prev items by using the swipe mechanism. That violates against WCAG success criterion "[2.5.1 Pointer Gestures](https://www.w3.org/WAI/WCAG22/quickref/#pointer-gestures)" and "[2.5.7 Dragging Movements](https://www.w3.org/WAI/WCAG22/quickref/#dragging-movements)".

### JS Framework

all

### Porsche Design System version

3.9.0

### What browsers are you seeing the problem on?

all

### Browser version

all

### Steps to reproduce the behavior

1. Go to [Carousel](https://designsystem.porsche.com/v3/components/carousel/examples) component with mobile device
2. Try to go to next/previous item without swiping (just by single touch)
3. See error (it's not possible)

### Expected behavior

All functionality that uses multipoint or path-based gestures for operation MUST be operable with a single pointer without a path-based gesture, unless a multipoint or path-based gesture is essential.

- [x] Solution: add possibility to navigate by pagination via click through carousel on mobile and desktop (pagination dots have to be at least 24x24 px)
  - infinite pagination case: consider pagination dots as next/prev buttons next to the centered one
  - none infinite pagination case: each pagination dot represents a specific slide 

### Additional improvements

- [x] `slot="header"` of Carousel shall always be placed left aligned independent of `alignHeader` prop
- [x] rename `slot="header"` to `slot="controls"`
- [x] evaluate why heading is needed to fullfil a11y compliance (remove or change validation if possible, so that heading is not needed to be set) - Validation can be removed and aria prop is recommended in docs when no heading is used.
- [x] add css variable `var(--p-carousel-prev-next-filter, none)` and document on Storefront which enables to invert the theme for prev next buttons by CSS filter value, e.g. `invert(1)` (https://porschedev.slack.com/archives/CT7AVHTTQ/p1704885069366889)

**Comment by michaelworm** [comment]
Currently blocked after discussing in the team by @stephanschroeter
